lib/components/Buttons/styles.ts

```diff
@@ -53,6 +51,7 @@ const buttonBase = css`
 export const buttonPrimary = css`
   ${buttonBase};
   background-color: var(--klima-green);
+  color: white; /* same in darkmode */
```

Comment:
So a new contrib @0xMakka was gonna tackle #880 as a first-ticket. Perhaps it's as easy as updating this line ahaha. Anyways ill leave that to him!

Reply:
To be fair there is a bit of selector wars going on so I doubt if this would be enough. See how he goes though.